satellite-software/src/BME680Sensor.cpp

```diff
@@ -303,8 +303,8 @@ const uint8_t* BME680Sensor::getCurrentMeasurementBuffer(void)
     PRINT_DEBUG(F("    calibrate temp = "));
     PRINT_DEBUG( temp_comp );
     PRINT_DEBUG(F(", normalize temp = "));
-    PRINT_DEBUG( (((float)temp_comp*9.0/5.0/100.0) + 32.0) );
-    PRINT_DEBUG(F(" °F, pressure = "));    
+    PRINT_DEBUG((float)temp_comp/100.0);
+    PRINT_DEBUG(F(" °C, pressure = "));    
```

Comment:
Sorry, I am not going to change the code from F to C. Can you make this a compile option instead? That way if the person wants the print out in F or C, they can chose. The default should be F because, well,  that's what I would make it.

Reply:
ah sorry, my bad, this was not supposed to be inside the pull request, it was meant for me for debugging, can you cherry pick the commit out of the pull request? or can i do it?

Reply:
I can't really cherry pick the pull request, and at this point, it would seem you have a lot of things in it that I won't merge because they pertain to your own personal configuration. Can you cancel this PR and try again? You should make a branch in your forked repository that represents what you are wanting to merge in with the main code.